tests/backend/realm/test_block.py

```diff
@@ -313,7 +325,7 @@ async def mock_read(organization_id, id):
 
     rep = await block_read(alice_ws, block)
     # A bad chunk result in a bad block, which should be detected by the client
-    assert rep == {"status": "ok", "block": ANY}
+    assert rep == BlockReadRepOk(BLOCK_DATA[:3] + bad_chunk[: len(BLOCK_DATA) - 3])
```

Comment:
use `isinstance(rep, BlockReadRepOk)` instead (the `BLOCK_DATA[:3] + bad_chunk[: len(BLOCK_DATA) - 3]` is complex to read and make the reader think this is a relevant part of the test while the test is about the fact that bad data in a blockstore doesn't cause a crash in the server)